test-suites/gutenberg/sanity-test-suites.md

```diff
@@ -172,6 +189,7 @@ This holds a grouping of certain test suites to run in order to share the work w
 - Cover 2
 - Social Icons 2
 - Image 2
+- Story block 1
```

Comment:
I noticed you added the Story block suite to each Test suite. The way we end up using these is we'll divide the work between each team member on Mobile-Gutenpages and run through one suite on iOS and one on Android (reference `Pb3aDo-fG`). So each release two suites are being run. 

We also try to keep each suite relatively even on the number of test cases. So I would suggest we add these all to either Test Suite 3 or 4 which have the fewest total test cases. 

Let me know if you feel strongly about having it run with each release and we can come up with a plan.

Reply:
Oh I see! I didn't know how these were actually used so, thanks for the explanation and pointers.

> Let me know if you feel strongly about having it run with each release and we can come up with a plan.

I think I want the tests to be run on each release because I don't want the block to disappear from the picker or misbehave from its basic working state from one release to the next one, or have a user report it before we notice, especially since we're doing a marketing effort soon with the launch of iOS next month.

Can we keep it this way for now? That said, if this is how things are rolling we can add all of these to Test Suite 3 or 4 as suggested which seem to have the fewest cases as you said, but only after a few releases have passed and things look to be stable enough cc @aforcier (I would strongly keep the sanity checks in their entirety although I can see how tedious it can get for ever single release). Certainly a topic of discussion, cc'ing @aforcier here as he might have other ideas / reasons as well.

Reply:
> only after a few releases have passed and things look to be stable enough

I can get behind running it with every release until we see some stability. I'm thinking we officially include it as part of Test Case 3. I could then add an `Additional Test suite(s)` column to the page referenced, and add this to each release for maybe 3 (🤷) releases after it ships. Then leave a note to re-assess if it should still be on the regular rotation. That might set up a good pattern for new blocks. 
WDYT?

Should this be run on each platform for each release or do we feel confident that platform differences will be caught by the other media upload tests?

Reply:
> I can get behind running it with every release until we see some stability. I'm thinking we officially include it as part of Test Case 3. I could then add an Additional Test suite(s) column to the page referenced, and add this to each release for maybe 3 (🤷) releases after it ships. Then leave a note to re-assess if it should still be on the regular rotation. That might set up a good pattern for new blocks.
WDYT?

Awesome! 🤝 :D done in d684657

> Should this be run on each platform for each release or do we feel confident that platform differences will be caught by the other media upload tests?

Thanks for asking this! Yes it's fine to run on one platform alone 👍